src/components/BookList.js

```diff
@@ -1,21 +1,99 @@
 import Book from './Book'
+import styled from 'styled-components/macro'
+import { useState } from 'react'
 
 function BookList({ books }) {
+  const [filteredBooks, setFilteredBooks] = useState(books)
+  const [readingStatus, setReadingStatus] = useState('')
+
+  function handleActiveReadingStatus(status, books) {
+    setReadingStatus(status)
+    if (status === 'true') {
+      const readBooks = books.filter(book => book.finished === true)
+      setFilteredBooks(readBooks)
+    } else if (status === 'false') {
+      const currentlyReadBooks = books.filter(book => book.finished === false)
+      setFilteredBooks(currentlyReadBooks)
```

Comment:
If `readingStatus` is a string, I would not recommend setting it to `'true'` or `'false'` as these are typically used as booleans. Maybe set the status to: `'finishedBooks'` and `'currentlyReading'`, that way it would be cleaner!